modules/react/tooltip/spec/Tooltip.spec.tsx

```diff
@@ -28,10 +28,9 @@ describe('Tooltip', () => {
       fireEvent.mouseEnter(screen.getByText('Test Text')); // triggers the tooltip
 
       jest.advanceTimersByTime(300); // advance the timer by the amount of delay time
-      expect(screen.getByText('Test Text')).toHaveAttribute('aria-describedby');
+      expect(screen.getByText('Test Text')).toHaveAttribute('aria-description');
 
-      const id = screen.getByText('Test Text').getAttribute('aria-describedby');
-      expect(screen.getByRole('tooltip')).toHaveAttribute('id', id);
+      expect(screen.getByRole('tooltip')).toHaveAttribute('id');
```

Comment:
I don't think it is important for the tooltip to have an `id` at this point. This line should be removed. The `id` was only used for the `aria-describedby`.